docker-compose/nested-spire/README.md

```diff
@@ -114,7 +114,8 @@ The `nestedA-server` must be registered on the `root-server` to obtain its ident
        -spiffeID "spiffe://example.org/nestedA" \
        -selector "docker:label:org.example.name:nestedA-server" \
        -downstream \
-       -ttl 3600
+       -x509SVIDTTL 3600 \
+       -jwtSVIDTTL 3600
```

Comment:
does this tutorial use JWT-SVIDs? If not, setting the TTL might be confusing.

Reply:
Not for this entry. I can remove it.